Makefile

```diff
@@ -1,30 +1,13 @@
 include Makefile.build.mk
-CURRENT_USER := $(if $(GITHUB_USER),$(GITHUB_USER),$(shell id -u -n))
```

Comment:
any reason to remove this? will it still work in actions?

Reply:
this was added for github codespaces, and that'll no longer be a supported dev env.